crypto/s2n_tls13_keys.c

```diff
@@ -343,3 +344,41 @@ int s2n_tls13_update_application_traffic_secret(struct s2n_tls13_keys *keys, str
 
     return 0;
 }
+
+int s2n_tls13_derive_resumption_master_secret(struct s2n_tls13_keys *keys, struct s2n_hash_state *hashes, struct s2n_blob *secret_blob)
```

Comment:
Nit: Alignment for this function seems a bit off, spacing needed before  `s2n_hash_state  *hashes`

Reply:
I am not seeing where the spacing is messed up, could you be more specific about the alignment issue?